### PR TITLE
Populate $userlevel from LDAP properly

### PR DESCRIPTION
**Pull request by jallakim**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

**Comment by jallakim** [comment]
Fixes #5090. Contributor agreement "signed" in PR #5091.

File: html/includes/authentication/ldap.inc.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ function get_userlevel($username)
 {
     global $config, $ldap_connection;
 
-    $userlevel = 0;
+    $userlevel = -1;
```
Comment (laf) [inline html/includes/authentication/ldap.inc.php]: Why -1?

Comment (jallakim) [inline html/includes/authentication/ldap.inc.php]: No specific reason. It was changed during my tests to rule out any mistakes. I can change it back to `0` if you want -- however, looking that how $userlevel is used, it doesn't matter if it's `0` or `-1`.

Comment (jallakim) [inline html/includes/authentication/ldap.inc.php]: I see that the other authentication methods has `0` as their value. For consistency, I'll change it back to `0`.

**Comment by laf** [comment]
image

**Pull request merged**